Sentinel/Classes/Core/Internal/Font+Size.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Font+Size.swift
+//  Pods
+//
+//  Created by Zvonimir Medak on 17.03.2025..
+//
+
+import Foundation
+import SwiftUI
+
+enum FontSize: CGFloat {
+    case title1 = 24
+    case subtitle1 = 16
+    case body1 = 14
+    case body2 = 13
+    case caption1 = 11
```

Comment:
Honestly, I would simplify that a bit; remove numbers and put only one `body`. It will make things easier to maintain and use.